.github/workflows/mvnw.yml

```diff
@@ -0,0 +1,25 @@
+name: Java CI
+
+#on:
```

Comment:
Intentionally commented? 
Will this run on every build without this?

Reply:
one of the last things to switch out before merging. it's still commented out for PR testing purposes